hazelcast/projection.py

```diff
@@ -0,0 +1,95 @@
+from hazelcast.serialization.api import IdentifiedDataSerializable
+
+_PROJECTIONS_FACTORY_ID = -30
+
+class Projection(object):
+    """Marker base class for all projections.
+
+    Projections allow the client to transform each query result object in order
+    to avoid redundant network traffic. The computation is performed in a fully
+    distributed manner, so no data other than the computed value is transferred
+    to the client, making the computation fast.
```

Comment:
```suggestion
    Projections allow the client to transform (strip down) each query result object 
    in order to avoid redundant network traffic.
```
The rest was not correct for the projections